apps/backend/src/db/migrations/20240821104015_create_categories_table.ts

```diff
@@ -0,0 +1,50 @@
+import { type Knex } from "knex";
+
+const TABLE_NAME = "categories";
+
+const ColumnName = {
+	CREATED_AT: "created_at",
+	ID: "id",
+	NAME: "name",
+	UPDATED_AT: "updated_at",
+} as const;
+
+const CATEGORIES = [
+	"Physical",
+	"Work",
+	"Friends",
+	"Love",
+	"Money",
+	"Free time",
+	"Spiritual",
+	"Mental",
+];
+
+async function up(knex: Knex): Promise<void> {
+	await knex.schema.createTable(TABLE_NAME, (table) => {
+		table.increments(ColumnName.ID).primary();
+		table.string(ColumnName.NAME).notNullable().unique();
+		table
+			.dateTime(ColumnName.CREATED_AT)
+			.notNullable()
+			.defaultTo(knex.fn.now());
+		table
+			.dateTime(ColumnName.UPDATED_AT)
+			.notNullable()
+			.defaultTo(knex.fn.now());
+	});
+
+	await knex(TABLE_NAME).insert(
```

Comment:
I would create 2 migrations - one for creating a table, one for adding data. Just for having everything separated